### DISCUSSION TITLE
[QDate] add prop to show previous and next months days that are selectable

### DISCUSSION THREAD

**Discussion by gregveres** [quasarframework/quasar#16632] [Ideas / Proposals]
The user experience of the date picker could be enhanced quite a bit by allowing the previous and next month days to be shown and selectable. 

Here is an image of what I mean:
![image](https://github.com/quasarframework/quasar/assets/12899823/cd892526-809a-4ff0-9d95-1811d297e0fc)
In the image, the date picker is showing the dates for September.

Since Sept 1 is on a Saturday, that first week on the current QDate would be showing 6 days of nothing.  And since Sept 30 is on Sunday, the last week on the current QDate would be showing 6 days of nothing. 

I would like to have a prop that enables those 6 days of nothing to show dates that can be selected. In the image, you see that the Aug 26 - 31st dates are displayed in a dimmed fashion, but they are still selectable. If the user selects Aug 28th, then the calendar would switch to Aug and would have Aug 28th selected. 

The same goes for Oct 1 - 6 in the last week. Those dates get shown in a dimmed state but they are selectable and if selcted, the calendar would switch to Oct with whichever date selcted as the selected date. 

I have used products with a date selector that functions like this and it is much nicer to use than the current workflow of QDate.

**Comment by moom-en** [comment]
Yes, I need this feature too

**Comment by TURPINJonathan** [comment]
I need it too

**Comment by zanzara** [comment]
Yes, it would be great to have this feature.
